common/djangoapps/student/views/management.py

```diff
@@ -1229,14 +1159,15 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
 
     if request.method == 'POST':
         password = request.POST['new_password1']
-        valid_link = False
-        error_message = validate_password_security_policy(user, password)
-        if not error_message:
-            # if security is not violated, we need to validate password
-            error_message = validate_password(password)
-            if error_message:
-                # password reset link will be valid if there is no security violation
-                valid_link = True
+        valid_link = True  # password reset link will be valid if there is no security violation
+        error_message = None
+        try:
+            validate_password(password, user=user)
+        except SecurityPolicyError as err:
+            error_message = err.message
+            valid_link = False
+        except ValidationError as err:
```

Comment:
Just want to double check this, so this will pass if the password during reset is invalid but secure. Is this method the one that resets the password or accepts the password on reset?